### PR TITLE
Make Dice index entry

### PR DESCRIPTION
**Pull request by DarthGandalf**
Bolden and linkify some of Dices

If player doesn't know what every die means, he may look at Dice entry in the index to read about them all

It would be best to add a photo of every die to the sections where they are explained

both-index
both-20
both-30
both-42

### CODE REVIEW

File: sections/all_map_locations.tex [hunk 1]
```diff
@@ -11,8 +11,8 @@ \subsection*{Symbols on the Map}
   \begin{tabularx}{\linewidth}{Y m{0.8\linewidth}}
     \small
     \mbox{\textbf{I-VII}} & Field Difficulty corresponding to the level of Neutral Units. \\
-    \includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus (\includesvg[height=10px]{\svgs/resource_die.svg}, \includesvg[height=10px]{\svgs/2_treasure_die.svg}, \includesvg[height=10px]{\svgs/experience.svg}, or \includesvg[height=10px]{\svgs/artifact.svg}).\index{Treasure Die} \\
-    \includesvg[height=15px]{\svgs/resource_die.svg} & Roll a Resource Die and gain the indicated Resources. \\
+    \includesvg[height=15px]{\svgs/treasure.svg} & Roll a \textbf{Treasure Die} and gain the indicated bonus (\includesvg[height=10px]{\svgs/resource_die.svg}, \includesvg[height=10px]{\svgs/2_treasure_die.svg}, \includesvg[height=10px]{\svgs/experience.svg}, or \includesvg[height=10px]{\svgs/artifact.svg}).\index{Treasure Die}\index{Dice} \\
```
Comment (qwrtln) [inline sections/all_map_locations.tex:14]:
```suggestion
    \includesvg[height=15px]{\svgs/treasure.svg} & Roll a \textbf{Treasure Die}\index{Treasure Die} and gain the indicated bonus (\includesvg[height=10px]{\svgs/resource_die.svg}, \includesvg[height=10px]{\svgs/2_treasure_die.svg}, \includesvg[height=10px]{\svgs/experience.svg}, or \includesvg[height=10px]{\svgs/artifact.svg}).\index{Treasure Die}\index{Dice} \\
```

File: sections/resources.tex [hunk 2]
```diff
@@ -34,7 +34,7 @@
 \centering\footnotesize\textcolor{darkcandyapplered}{\textbf{\textit{Resource Production Tracker}}}
 \bigskip
 
-Possible Resource Die\index{Resource Die} \includesvg[height=10px]{\svgs/resource_die.svg} results:
+Possible \textbf{Resource Die}\index{Resource Die}\index{Dice} \includesvg[height=10px]{\svgs/resource_die.svg} results:
```
Comment (qwrtln) [inline sections/resources.tex:37]: I like the index entry but I don't think `textbf` is necessary.

Comment (DarthGandalf) [reply sections/resources.tex:37]: In theory I agree. But if I'm coming here via the link from Index if I want to see the die, it's easier to spot the die if it's bold. But having an image of the die would be better than textbf. I just don't have the image - would need to find one somewhere...

Comment (qwrtln) [reply sections/resources.tex:37]: Fair. Let me take a look.

File: sections/resources.tex [hunk 3]
```diff
@@ -46,6 +46,11 @@
   \item \includesvg[height=15px]{\svgs/6_gold.svg} 6 x Gold
 \end{itemize}
 
+% \includegraphics[width=0.3\linewidth]{\images/resource_die.png}
```
Comment (qwrtln) [inline sections/resources.tex:49]:
```suggestion
```
My leftover.

File: sections/resources.tex [hunk 4]
```diff
@@ -46,6 +46,10 @@
   \item \includesvg[height=15px]{\svgs/6_gold.svg} 6 x Gold
 \end{itemize}
 
+\begin{tikzpicture}[overlay]
+  \node at (2, 2) {\includegraphics[width=0.3\linewidth]{\images/resource_die.png}};
```
Comment (qwrtln) [inline sections/resources.tex:50]:
```suggestion
  \node at (2, 1.5) {\includegraphics[width=0.3\linewidth]{\images/resource_die.png}};
```
IMO, that's a little too close to the text.

**Comment by DarthGandalf** [comment]
Added all the dice images

**Review by qwrtln — CHANGES_REQUESTED** [review]
Run `po4a` and it's a ✅

**Review by qwrtln — APPROVED** [review]

**Review by evilmint — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**